lib/driver.js

```diff
@@ -434,7 +461,13 @@ class XCUITestDriver extends BaseDriver {
         });
         this.logEvent('wdaSessionStarted');
       } catch (err) {
-        return await quitAndUninstall(`Unable to start WebDriverAgent session: ${err.message}`);
+        let errorMsg = `Unable to start WebDriverAgent sesssion because of xcodebuild failure: ${err.message}`;
```

Comment:
error messages are different for both cases. I separated them intentionally for a case it will be necessary to add any other helpful message details to help user to fix his problem for the particular case.